Makefile

```diff
@@ -6,6 +6,8 @@ ifeq ($(OS),Windows_NT)
 else
   UNAME_S := $(shell uname -s)
 
+  LINKER_FLAGS = -fuse-ld=gold
```

Comment:
This flag isn't available on MacOS, and I don't think it's available on FreeBSD. It's being set for anything that isn't Windows.